docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
```

Comment:
I think that we can live without docker compose? Since the use case that we primarily forsee is simln dropping into other people's existing setups (they'll add it to _their_ docker compose. 

For including a volume we can just add instructions to include a volume with docker run?